### PR TITLE
Added documentation for to the theme/logo customization feature

### LINKED ISSUE #566: Add documentation for the Icon/Theme customization feature
### What do you want to improve?
Add documentation on how to use the Icon/Theme customization feature added in https://github.com/janus-idp/backstage-showcase/pull/555 and https://github.com/janus-idp/backstage-showcase/pull/564
### What is the current documentation?
There is currently no documentation for this feature
### What is the new documentation?
Documentation that describes how to use the feature and what are the supported input format.

### PR DESCRIPTION
**Pull request by Zaperex**
## Description
Added initial documentation related to the theme/logo customization features 

## Which issue(s) does this PR fix

- Fixes #566

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [x] Documentation is updated if necessary (requirement for new features)
- [x] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by gorkem — APPROVED** [review]
lgtm

File: .changeset/ten-plums-flow.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (invincibleJai) [inline .changeset/ten-plums-flow.md:1]: will this be empty?

File: showcase-docs/customization.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- [color()](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color)
+
+### Customizing the primary colors
+
+To customize the color of links and buttons, provide your color in a [supported format](#customizing-the-your-showcase-colors) to the `app.branding.theme.[light or dark].primaryColor` field of the `app-config.yaml`:
+
+```yaml
+app:
+  branding:
+    theme:
```
Comment (invincibleJai) [inline showcase-docs/customization.md:60]: should we mention these are like theme IDs as comment or note?

File: .changeset/ten-plums-flow.md (hunk 1 above)
Comment (Zaperex) [reply .changeset/ten-plums-flow.md:1]: Yes, I just added it to pass the CI cases. I can say it's a patch change for app to add documentation if needed.

File: showcase-docs/customization.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      light:
```
[the remark this answers is not in the document]
Comment (Zaperex) [reply showcase-docs/customization.md:61]: Ya I think I can mention that the themes for light mode and dark mode are separate and will require separate configurations.

File: .changeset/ten-plums-flow.md (hunk 1 above)
Comment (schultzp2020) [reply .changeset/ten-plums-flow.md:1]: @Zaperex Please only add one changeset.

Comment (Zaperex) [reply .changeset/ten-plums-flow.md:1]: Ah my mistake, I thought I was editting the old one

Comment (Zaperex) [reply .changeset/ten-plums-flow.md:1]: Fixed

**Review by invincibleJai — APPROVED** [review]

**Review by christophe-f — APPROVED** [review]

**Pull request merged**